backend/src/main/java/com/soen341/instagram/controller/ProfileController.java

```diff
@@ -0,0 +1,92 @@
+package com.soen341.instagram.controller;
+
+// Spring Boot
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+// Project
+import com.soen341.instagram.dto.account.ProfileRequestDTO;
+import com.soen341.instagram.service.impl.ProfileService;
+
+@RestController
+public class ProfileController {
+	
+	@Autowired
+	private ProfileService profileService;
+	
+// For now we are not allowing the user to change their username
```

Comment:
Not sure how we want to do it for this project, but at work all that was "TODO" or temporary things, we didn't add/comment out until we needed to implement it (and hence create a new commit for it). Can I ask why we aren't implementing this right now? (the same applies for the commented code in `ProfileService`)

Reply:
So if we were to implement this, the code would stay the same so I didn't think it would be useful to delete it just to rewrite it. The problem here is that the username is used as the primary key in the database and changing it causes some issues. There are ways around this that can be done, but as discussed offline with Simon yesterday, some decisions have to be made as to the best way to do this or if it is worth doing at all.
A similar case is to be made for the picture, the code would stay very similar, however, we came to realize that the current method for uploading profile pictures needs to be refactored and this should wait until then.